.github/workflows/publish_test_package.yml

```diff
@@ -0,0 +1,71 @@
+name: Publish test package
+on: 
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  publish-test-package:
+    # This action only runs on forks because it pushes a new release.
+    # It is not wanted on the main repository as it would spam the releases.
+    if: ${{ github.repository != 'canalplus/rx-player' }}
+
+    runs-on: [ubuntu-latest]
+    permissions:
+      packages: write
+      contents: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 20
+          scope: "@${{ github.repository_owner }}"
+          registry-url: "https://npm.pkg.github.com"
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+ 
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build Library
+        run: npm run build
+
+      - name: Create Zip file
+        run: echo "npm-pack=$(npm pack)" >> $GITHUB_OUTPUT
+        id: npm-pack
+
+      - run : "echo ${{ github.sha }}"
+      
+      - name: Upload Zip to release
```

Comment:
I think technically this is not a "Zip", as the zip format as I understand it are `.zip` files which may contain multiple files.

Here that `.tgz` most likely rely on the gzip format instead (which also rely on the DEFLATE algorithm for compression but do it in a single-file-way), with then `tar` used to create an archive containing multiple files.

We can just say archive I think here.